frame/contracts/src/wasm/prepare.rs

```diff
@@ -148,10 +148,10 @@ impl<'a> ContractModule<'a> {
 	fn inject_gas_metering(self) -> Result<Self, &'static str> {
 		let gas_rules =
 			rules::Set::new(
-				self.schedule.regular_op_cost.clone().saturated_into(),
+				self.schedule.op_cost_regular.clone().saturated_into(),
```

Comment:
If schedule changes, those injected gas metering get out of date no ?
(I know it is not introduced by this PR)

Reply:
That is correct. We re-instrument any code on contract call when the `Schedule` was changed since the last call. The `Schedule` is versioned for that reason.